src/lib/autoinstall/clients/clone_system.rb

```diff
@@ -92,9 +92,15 @@ def main
               "filename" => {
                 "type" => "string",
                 "help" => "filename=OUTPUT_FILE"
+              },
+              "target"   => {
+                "type" => "string",
+                "help" => _(
+                  "How much information to include in the profile ('default' or 'compact')"
```

Comment:
Well, two thinks here. I think it deserve a bit more explanation to users and also CmdLine support enums - https://github.com/yast/yast-yast2/blob/7fc264a1dcb28c175d5fb46d1f4519221fd368f4/library/commandline/src/modules/CommandLine.rb#L1468